File: GSI QA testing tool/UI/UI_012_EqualOpportunity.cs
using GSI_QA_testing_tool.Utilities;
using OpenQA.Selenium;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GSI_QA_testing_tool.UI
{
    internal class UI_012_EqualOpportunity
    {
        public static void GoTo(IWebDriver driver)
        {

            if (Data._StopAt.Contains("Equal Opportunity"))
            {
                CustomDialog customDialog = new CustomDialog("Make Selection", "Select pages you want to stop at, make changes and click ok before clicking the Next button");
                customDialog.ShowDialog();
            }



            if (Finder.FindIt(driver, "//input[@id='ctl00_Main_content_btnAgree']"))
            {
                Debug.WriteLine("EqualOpportunity is On");
            }
            else 
            {
                Debug.WriteLine("EqualOpportunity is Off");
                return;
            }



            Finder.ClickIt(driver, "//input[@id='ctl00_Main_content_btnAgree']");
        }
    }
}

File: GSI QA testing tool/UI/UI_012_EqualOpportunity.vb
Imports OpenQA.Selenium
Imports System
Imports System.Collections.Generic
Imports System.Diagnostics
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace GSI_QA_testing_tool.UI
    Friend Class UI_012_EqualOpportunity
        Public Shared Sub [GoTo](ByVal driver As IWebDriver, ByVal js As IJavaScriptExecutor)
            If Data._StopAt.Contains("Equal Opportunity") Then
                Data.StopAtDialog()
            End If

            If Finder.FindItByCSS(driver, "#ctl00_Main_content_btnAgree") Then
                Debug.WriteLine("EqualOpportunity is On")
            Else
                Debug.WriteLine("EqualOpportunity is Off")
                Return
            End If

            js.ExecuteScript("arguments[0].click();", driver.FindElement(By.XPath("//input[@id='ctl00_Main_content_btnAgree']")))
        End Sub
    End Class
End Namespace
